doc/quickstart/mapbender_quickstart.rst

```diff
@@ -331,96 +168,6 @@ Try it yourself
-
-
-Create a user
-================================================================================
-
-#. To create a user go to :guilabel:`New User` or click the :menuselection:`+-Button`.
-
-#. Choose a name and a password for your user.
-
-#. Provide an email address for the user.
-
-#. Save your new user.
-
-#. You can provide more information about the user in the tab :menuselection:`Profile`.
-
-.. image:: /images/projects/mapbender/mapbender3_create_user.png
-     :scale: 70 %
-
-
-Create a group
-================================================================================
-#. Create a group by :guilabel:`New Group`.
-
-#. Define a name and a description for your group.
-
-#. Save your new group.
-
-
-Assign users to group
-================================================================================
-
-#. Assign a user to a group by :guilabel:`Users --> Groups`.
-
-#. Choose one or more users you want to add to the group at :menuselection:`Users`.
-
-#. Assign a user by :menuselection:`Users --> Edit-Button--> Groups` to a group.
-
-  .. image:: /images/projects/mapbender/mapbender3_assign_user_to_group.png
-     :scale: 70 %
-
-
-Roles
-================================================================================
-Mapbender provides different rights. They refer to the Symfony ACL System http://symfony.com/doc/2.8/cookbook/security/acl_advanced.html#built-in-permission-map
-
-* view - Whether someone is allowed to view the object.
-* edit - Whether someone is allowed to make changes to the object.
-* delete - Whether someone is allowed to delete the object.
-* operator - Whether someone is allowed to perform all of the above actions.
-* master - Whether someone is allowed to perform all of the above actions, and in addition is allowed to grant any of the above permissions to others.
-* owner - Whether someone owns the object. An owner can perform any of the above actions and grant master and owner permissions.
-
-#. Assign roles to a user by :menuselection:`Users --> Edit your User --> Security`.
-
-  .. image:: /images/projects/mapbender/mapbender3_roles.png
-     :scale: 70 %
-
-
-Assign an Application to a User/Group
-================================================================================
-#. Edit your application by :menuselection:`Application --> Edit-Button`.
-
-#. Choose :menuselection:`Security`
-
-#. Set permission like view edit delete operator master owner
-
-#. Assign a user/group to the application
-
-#. Test your configuration!
-
-#. Logout from Mapbender by :menuselection:`Logout`.
-
-#. Login as the new user
-
-  .. image:: /images/projects/mapbender/mapbender3_security.png
-     :scale: 70 %
-
-
-Things to try
-================================================================================
-
 Here are some additional challenges for you to try:
 
 #. Try to load some WMS in your application. Try to configure your WMS.
```

Comment:
add more challenges like
* Create users and groups and assign an application to the new user/group

Reply:
done.